lib/clients/index.js

```diff
@@ -0,0 +1,57 @@
+function MetricClients(backends) {
+  this.clients = [];
+  if (backends) {
+    this.init(backends);
+  }
+}
+
+/**
+ * Init clients for each of the backend specified in the given backends
+ * 
+ * @param {Array} backends the backend configurations
+ */
+MetricClients.prototype.init = function(backends) {
+  var self = this;
+  if (backends) {
+    backends.forEach(function(backendConfig){
+      var client = self.initClient(backendConfig);
+      if (client) {
+        self.clients.push(client);
+      }
+    });
+  }
+};
+
+/**
+ * Init a single client for the given backend
+ * 
+ * @param {Object} backend
+ */
+MetricClients.prototype.initClient = function(backend) {
+  var self = this;
+  var type = backend.type;
+  var clientModule;
+  try  {
+    clientModule = require('./' + type);
+  } catch (e) {
+    console.warn('can not find valid client for metrics backend ' + JSON.stringify(backend));
+  }
+  if (clientModule) {
+    return clientModule.init(backend);
+  } else {
+    return null;
+  }
+};
+
+/**
+ * Send messages. This will invoke the `send` method of each client
+ * 
+ * @param {Object} message 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)